### DISCUSSION TITLE
Student registration Issue

### DISCUSSION THREAD

**Discussion by mskvofficial** [community/community#121392] [GitHub Education]
### Hi everyone,

I had applied for student verification in GitHub then they asked me to upload my College ID Card. after completing the work they informed me that you should enable the Two factor authentication and billing work then  i completed the work then again i uploaded the ID card but shows like this. what can i do and how can i get student Verification ?

<img width="950" alt="Screenshot 2024-04-28 at 9 30 32 AM" src="https://github.com/community/community/assets/85546952/751c2f61-738e-4b6c-954c-1898259dec18">

**Comment by haululazkiyaa** [comment]
have you try with another documents, except ID Card???

**Reply by mskvofficial** [reply]
not yet I'll try it now

**Comment by samus-aran** [comment]
Hi @iamsurya786 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Comment by itzzjb** [comment]
You can try the same document but a new image which is more clearer than the previous one. It's good if you can crop the image right along with the document boarder. Also remember to use a jpeg or png file.

Cheers!
